test/versioned/connect/route.tap.js

```diff
@@ -28,7 +28,7 @@ test('transaction tests', function (t) {
     t.plan(10)
     const connect = require('connect')
     const { version: pkgVersion } = require('connect/package')
-    let server
+    let server = null
```

Comment:
Is this change cosmetic? The variable never seems to be used or checked before it's assigned a non-null value below.

Reply:
good question, most of this was from michael. I can dig

Reply:
it's prob a habit he gets in. no harm in this, I'm going to keep